src/docs/product/sentry-basics/tracing/trace-view.mdx

```diff
@@ -33,7 +33,7 @@ In a fully instrumented system, a span in one service will connect to a transact
 In addition, broken subtraces can occur when Sentry receives an error for a trace, but the corresponding transaction does not exist. In this case, the error will not appear on the trace view and the trace navigator won't appear on the **Issue Details** page. In these cases you can click "Open In Discover" to see all the events in the trace.
 
 Broken subtraces may be caused by:
-- Sample rates may prevent the SDK from sending a transaction, so not all errors will have an associated transaction in Sentry.
+- [Turn on Server-side Sampling](/product/data-management-settings/server-side-sampling/). Client-side sampling may prevent the SDK from sending a transaction, server-side sampling gives you greater control over which Transactions are stored and processed.
```

Comment:
Hopefully I haven't changed the meaning here, but we definitely want to clean up the wording and sentence fragments that were here before.

```suggestion
- [Server-side sampling](/product/data-management-settings/server-side-sampling/)  isn't enabled. Client-side sampling may prevent the SDK from sending a transaction. Turn on server-side sampling to give you greater control over which transactions are processed and indexed.
```